frontends/web/src/locales/en/app.json

```diff
@@ -1530,6 +1530,7 @@
     "coincontrol": {
       "address": "Address",
       "addressReused": "Address re-used",
+      "change": "Change",
```

Comment:
hm not sure if this needs to be translated or if we just keep the label as "change", but no strong opinion.

Reply:
I think keeping a label makes sense, in case we want to translate. We can always keep the label untranslated across languages.

Reply:
the other day we had an error something about WC namespace which we found out it makes more sense to not translate. But translators might miss that and translate anyway.

Reply:
asked a bit around, people would like to see it translated ie. "Wechselgeld" in German.

lets keep as is.